### PR TITLE
Fixed remaining linting errors

### PR DESCRIPTION
**Pull request by tboerger**
* [x] Fix general linting errors
* [x] Fix bindata linting errors
* [x] Integrate lint check into drone

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -131,6 +131,8 @@ bindata: modules/bindata/bindata.go
 modules/bindata/bindata.go: $(BINDATA)
 	go-bindata -o=$@ -ignore="\\.go|README.md|TRANSLATORS" -pkg=bindata conf/...
 	go fmt $@
+	sed -i.bak 's/confLocaleLocale_/confLocaleLocale/' $@
```
Comment (bkcsoft) [inline Makefile:134]: Change to `sed -i '' 's/......`, works on linux,freebsd,macos :)

**Review by bkcsoft — CHANGES_REQUESTED** [review]
One little thing

**Comment by Bwko** [comment]
Except for ^ LGTM

File: Makefile (hunk 1 above)
Comment (tboerger) [reply Makefile:134]: Nope, that won't work on linux, at least not on Alpine.

```
sed -i '' 's/confLocaleLocale_/confLocaleLocale/' modules/bindata/bindata.go
sed: s/confLocaleLocale_/confLocaleLocale/: No such file or directory
```

**Review by bkcsoft — APPROVED** [review]

**Comment by bkcsoft** [comment]
`sed` apparently isn't cross-platform friendly so screw it. LGTM

**Pull request merged**